include/wx/generic/private/grid.h

```diff
@@ -1005,5 +1005,12 @@ class WXDLLIMPEXP_ADV wxGridTypeRegistry
     wxGridDataTypeInfoArray m_typeinfo;
 };
 
+// Returns the rect of the check box in a cell with the given alignmens
+// and the size.
+// The function is used by wxGridCellBoolEditor and wxGridCellBoolRenderer.
+wxRect GetGridCheckBoxRect(const wxSize& checkBoxSize,
```

Comment:
```suggestion
wxRect wxGetGridCheckBoxRect(const wxSize& checkBoxSize,
```

Even if it's a private function, it's better to still prefix it with `wx` if only for clarity. Another alternative would be to use `wxImpl` or `wxPrivate` namespace, but it's probably not worth it for a single function.